storage-account-ld-location.tf

```diff
@@ -0,0 +1,66 @@
+locals {
+  product                   = "ldlocation"
+  ld_location_account_name  = "${local.product}${var.env}"
+  container_name            = "lrd-ref-data"
+  container_archive_name    = "lrd-ref-data-archive"
+}
+
+module "storage_account_ld_location" {
+  source                   = "git@github.com:hmcts/cnp-module-storage-account?ref=master"
+  env                      = "${var.env}"
+  storage_account_name     = "${local.ld_location_account_name}"
+  resource_group_name      = "${azurerm_resource_group.rg.name}"
+  location                 = "${var.location}"
+  account_kind             = "BlobStorage"
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+  access_tier              = "Hot"
```

Comment:
lets keep replication as ZRS for some redundancy

Reply:
Should we consider having ZRS for prod and LRS for lower environments to save cost?

Reply:
yes thats better!!

Reply:
Have made the changes, pls review and approve.